src/realm/table_view.cpp

```diff
@@ -356,9 +375,9 @@ double TableViewBase::minimum_double(size_t column_ndx, size_t* return_ndx) cons
 OldDateTime TableViewBase::minimum_olddatetime(size_t column_ndx, size_t* return_ndx) const
 {
     if (m_table->is_nullable(column_ndx))
-        return aggregate<act_Max, int64_t>(&IntNullColumn::minimum, column_ndx, 0, return_ndx);
+        return aggregate<act_Min, int64_t>(&IntNullColumn::minimum, column_ndx, 0, return_ndx);
     else
-        return aggregate<act_Max, int64_t>(&IntegerColumn::minimum, column_ndx, 0, return_ndx);
+        return aggregate<act_Min, int64_t>(&IntegerColumn::minimum, column_ndx, 0, return_ndx);
```

Comment:
If the TableView size matched the Table size, the bug got bypassed by executing the correct "min" method directly on the Column. This fix is irrelevant today though because this old datetime code is unreachable except through unit test